### PR TITLE
[INJIMOB-698]: FAQ additions for Activation of card & Retriving ID related processes.

### PR DESCRIPTION
**Pull request by Anil-kumar-Majji**
**[INJIMOB](https://mosip.atlassian.net/browse/INJIMOB-698)** :- FAQ additions related to Activation of card & Retriving ID

### CODE REVIEW

File: components/HelpScreen.tsx [hunk 1]
```diff
@@ -31,39 +57,102 @@ export const HelpScreen: React.FC<HelpScreenProps> = props => {
...
+            <Text style={Theme.TextStyles.helpDetails}>{t('detail-10')}</Text>
+            <Text margin="7" style={Theme.TextStyles.header}>
+              {t('whatAreTheDifferentTypesOfId?')}
+            </Text>
+            <Text style={Theme.TextStyles.helpDetails}>
+              {t('detail-11')}
+              {hyperLinkString(
+                'here',
+                diffTypesOfIdsUrl,
+                '1.2.0/id-lifecycle-management/identifiers',
```
Comment (swatigoel) [inline components/HelpScreen.tsx:71]: Where is it redirecting?

File: components/HelpScreen.tsx [hunk 2]
```diff
@@ -1,15 +1,41 @@
...
+  var diffTypesOfIdsUrl = hereUrl.slice(0, -4);
+
+  const hyperLinkString = (
+    word: string,
+    urlString: string,
+    extension?: string,
+  ) => {
+    return (
+      <Text
+        style={{color: Theme.Colors.urlLink, fontFamily: 'Inter_600SemiBold'}}
```
Comment (vijay151096) [inline components/HelpScreen.tsx:31]: extract the theme to respective theme files

File: components/HelpScreen.tsx [hunk 3]
```diff
@@ -1,15 +1,41 @@
...
   const [showHelpPage, setShowHelpPage] = useState(false);
+  const [hereUrl, setHereUrl] = useState('');
+
+  useEffect(() => {
+    getAllConfigurations().then(response => {
+      setHereUrl(response.aboutInjiUrl);
+    });
+  }, []);
+
+  var diffTypesOfIdsUrl = hereUrl.slice(0, -4);
```
Comment (swatigoel) [inline components/HelpScreen.tsx:22]: Which url we are trying to achieve here?

File: components/HelpScreen.tsx [hunk 4]
```diff
@@ -31,39 +57,102 @@ export const HelpScreen: React.FC<HelpScreenProps> = props => {
         <ScrollView>
           <Column fill padding="10" align="space-between">
             <Text margin="7" style={Theme.TextStyles.header}>
-              {t('whatIsDigitalCredential?')}
+              {t('whatIsAnId?')}
+            </Text>
+            <Text style={Theme.TextStyles.helpDetails}>{t('detail-10')}</Text>
+            <Text margin="7" style={Theme.TextStyles.header}>
```
Comment (vijay151096) [inline components/HelpScreen.tsx:63]: can we add the margin inside the styles itself

File: components/HelpScreen.tsx [hunk 5]
```diff
@@ -31,39 +57,102 @@ export const HelpScreen: React.FC<HelpScreenProps> = props => {
...
+            </Text>
+            <Text style={Theme.TextStyles.helpDetails}>
+              {t('detail-11')}
+              {hyperLinkString(
+                'here',
+                diffTypesOfIdsUrl,
+                '1.2.0/id-lifecycle-management/identifiers',
+              )}
+            </Text>
+            <Text margin="7" style={Theme.TextStyles.header}>
```
Comment (vijay151096) [inline components/HelpScreen.tsx:74]: same here as well
https://github.com/mosip/inji/pull/1213/files#r1470853503

File: components/HelpScreen.tsx (hunk 2 above)
Comment (Anil-kumar-Majji) [reply components/HelpScreen.tsx:31]: Done.

File: components/HelpScreen.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (swatigoel) [reply components/HelpScreen.tsx:22]: +1 to @vijay151096

File: components/HelpScreen.tsx [hunk 6]
```diff
@@ -1,15 +1,35 @@
...
 import {ScrollView} from 'react-native-gesture-handler';
 import {Column, Text} from './ui';
 import {Theme} from './ui/styleUtils';
+import getAllConfigurations from '../shared/commonprops/commonProps';
 
 export const HelpScreen: React.FC<HelpScreenProps> = props => {
   const {t} = useTranslation('HelpScreen');
 
   const [showHelpPage, setShowHelpPage] = useState(false);
+  const [hereUrl, setHereUrl] = useState('');
```
[the remark this answers is not in the document]
Comment (Anil-kumar-Majji) [reply components/HelpScreen.tsx:14]: I have modified in to "injiHelpUrl".

File: components/HelpScreen.tsx (hunk 1 above)
Comment (Anil-kumar-Majji) [reply components/HelpScreen.tsx:71]: Now I changed this name to "**customisedUrl**". It navigates to INJI life-cycle managements  [identifiers](https://docs.mosip.io/1.2.0/id-lifecycle-management/identifiers)

File: components/HelpScreen.tsx (hunk 3 above)
Comment (Anil-kumar-Majji) [reply components/HelpScreen.tsx:22]: Now I changed this name to "**customisedUrl**". It navigates to MOSIP id-life-cycle managements  [identifiers](https://docs.mosip.io/1.2.0/id-lifecycle-management/identifiers)

File: components/HelpScreen.tsx (hunk 5 above)
Comment (Anil-kumar-Majji) [reply components/HelpScreen.tsx:74]: No. We can't

File: components/HelpScreen.tsx (hunk 4 above)
Comment (Anil-kumar-Majji) [reply components/HelpScreen.tsx:63]: header in the TextStyles itself is an object from Themes. Here we can't. I suggest adding a margin in the header object, leading to changes for other heading in the app.

Comment (vijay151096) [reply components/HelpScreen.tsx:63]: Then Lets add this style as separate style in themes and include that style in the array.

File: components/HelpScreen.tsx (hunk 5 above)
Comment (vijay151096) [reply components/HelpScreen.tsx:74]: Then Lets add this style as separate style in themes and include that style in the array.

File: components/HelpScreen.tsx (hunk 4 above)
Comment (Anil-kumar-Majji) [reply components/HelpScreen.tsx:64]: Done @vijay151096

File: components/HelpScreen.tsx (hunk 5 above)
Comment (Anil-kumar-Majji) [reply components/HelpScreen.tsx:75]: Done.

**Review by swatigoel — APPROVED** [review]

**Review by swatigoel — APPROVED** [review]

**Pull request merged**